tests/nested/manual/muinstaller-core/task.yaml

```diff
@@ -217,11 +215,12 @@ execute: |
   {
     "snaps": ["snap-with-comps"],
     "components": {
-      "snap-with-comps": ["comp1"];
```

Comment:
Huh? Was this masked before somehow?

Reply:
yup, must have . I have no idea how the test was passing in the first place:
```
maciek@galeon:work/canonical/snapd (git)-[bboozzoo/fde-dbx-state-wip] cat expected-available.json
{
  "snaps": ["snap-with-comps"],
  "components": {
    "snap-with-comps": ["comp1"];
  }
}
maciek@galeon:work/canonical/snapd (git)-[bboozzoo/fde-dbx-state-wip] jq --sort-keys expected-available.json
jq: error: expected/0 is not defined at <top-level>, line 1:
expected-available.json
jq: error: available/0 is not defined at <top-level>, line 1:
expected-available.json         
jq: 2 compile errors
maciek@galeon:work/canonical/snapd (git)-[bboozzoo/fde-dbx-state-wip] echo $?
3
maciek@galeon:work/canonical/snapd (git)-[bboozzoo/fde-dbx-state-wip] 
```